frontend/src/features/public-form/components/FormPaymentPage/stripe/components/PaymentBlock.tsx

```diff
@@ -181,7 +184,6 @@ export const StripePaymentBlock = ({
             S${centsToDollars(form?.payments_field?.amount_cents || 0)}
           </Text>
         </Text>
-
```

Comment:
eeyer more of your LOC changes HAHA

Reply:
Shhhh!

Reply:
Probably I tried removing `<Element />` somewhere and realized that it looked better back here 🤔